pkgs/os-specific/linux/sssd/default.nix

```diff
@@ -18,6 +22,7 @@ stdenv.mkDerivation rec {
     url = "https://fedorahosted.org/released/sssd/${pname}-${version}.tar.gz";
     sha256 = "0ngr7cgimyjc6flqkm7psxagp1m4jlzpqkn28pliifbmdg6i5ckb";
   };
+  patches = [ patch-samba_4_12 ];
```

Comment:
It is probably cleaner to put the `fetchpatch` invocation directly to `patches`. Also a comment would be nice. Example:

https://github.com/NixOS/nixpkgs/blob/46ec52f32966f16ce2bdd12bd150841e058988e9/pkgs/desktops/gnome-3/core/gnome-shell/default.nix#L83-L88

Reply:
Done.